Xamarin.Forms.Platform.UAP/ActivityIndicatorRenderer.cs

```diff
@@ -1,9 +1,13 @@
 ﻿using System.ComponentModel;
+
 using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+using Xamarin.Forms.PlatformConfiguration.WindowsSpecific;
 
 namespace Xamarin.Forms.Platform.UWP
 {
-	public class ActivityIndicatorRenderer : ViewRenderer<ActivityIndicator, FormsProgressBar>
+	public class ActivityIndicatorRenderer : ViewRenderer<ActivityIndicator, FrameworkElement>
```

Comment:
breaking change

Reply:
actually it is not breaking existing apps. only type became dynamic. ForsProgresBar implements a FrameworkElement. ProgressRing as well. So in order to SetNativeControl function based on chosen type, this had to be changed to FrameworkElement. But it surely doesnt break any change. If existing users are having "Bar" which is default when it is not set, i will continue using "Bar" style.